arangod/RestServer/EndpointFeature.cpp

```diff
@@ -58,23 +58,132 @@ void EndpointFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
+2022-11-07T10:39:30Z [1] INFO [6ea38] {general} using endpoint 'http+tcp://0.0.0.0:8529' for non-encrypted requests
+2022-11-07T10:39:31Z [1] INFO [cf3f4] {general} ArangoDB (version 3.10.0 [linux]) is ready for business. Have fun!
+```
+
+On one specific ethernet interface, each port can only be bound
+**exactly once**. You can look up your available interfaces using the `ifconfig`
+command on Linux / macOS, and `ipconfig` on Windows. The general names of the
+interfaces differ between operating systems and the hardware they run on.
+However, every host has typically a so called loopback interface, which is a
+virtual interface. By convention, it always has the address `127.0.0.1` (IPv4)
+or `::1` (IPv6), and can only be reached from the very same host. Ethernet
+interfaces usually have names like `eth0`, `wlan0`, `eth1:17`, `le0`, or a
+plain text name in Windows.
+
+To find out which services already use ports (so ArangoDB can't bind them
+anymore), you can use the `netstat` command. It behaves a little different on
+each platform; run it with `-lnpt` on Linux, `-p tcp` on macOS, or with `-an`
+on Windows for valuable information.
+
+ArangoDB can also do a so called *broadcast bind* using `tcp://0.0.0.0:8529`.
+This way, it is reachable on all interfaces of the host. This may be useful on
+development systems that frequently change their network setup, like laptops.
+
+ArangoDB can also listen to IPv6 link-local addresses via adding the zone ID
+to the IPv6 address in the form `[ipv6-link-local-address%zone-id]`. However,
+what you probably want instead is to bind to a local IPv6 address. Local IPv6
+addresses start with `fd`. If you only see a `fe80:` IPv6 address in your
+interface configuration but no IPv6 address starting with `fd`, your interface
+has no local IPv6 address assigned. You can read more about IPv6 link-local
+addresses here: https://en.wikipedia.org/wiki/Link-local_address#IPv6.
+
+To bind to a link-local and local IPv6 address, run `ifconfig` or equivalent
+command. The command lists all interfaces and assigned IP addresses. The
+link-local address may be `fe80::6257:18ff:fe82:3ec6%eth0` (IPv6 address plus
+interface name). A local IPv6 address may be `fd12:3456::789a`.
+To bind ArangoDB to it, start `arangod` with
+`--server.endpoint tcp://[fe80::6257:18ff:fe82:3ec6%eth0]:8529`.
+Use `telnet` to test the connection.
+
+```bash
+> telnet fe80::6257:18ff:fe82:3ec6%eth0 8529
+Trying fe80::6257:18ff:fe82:3ec6...
+Connected to my-machine.
+Escape character is '^]'.
+GET / HTTP/1.1
+
+HTTP/1.1 301 Moved Permanently
+Location: /_db/_system/_admin/aardvark/index.html
+Content-Type: text/html
+Server: ArangoDB
+Connection: Keep-Alive
+Content-Length: 197
+
+<html><head><title>Moved</title></head><body><h1>Moved</h1><p>This page has moved to <a href="/_db/_system/_admin/aardvark/index.html">/_db/_system/_admin/aardvark/index.html</a>.</p></body></html>
```

Comment:
Seems superfluous here and too detailed. We shouldn't care too much about telnet's output in our documentation.